lib/guest_installation_and_configuration_base.pm

```diff
@@ -1219,7 +1230,13 @@ sub start_guest_installation {
     script_run("set +o pipefail");
     save_screenshot;
     record_info("Guest $self->{guest_name} installation dry run succeeded", "Going to install by using $self->{virt_install_command_line}");
-    type_string("screen -t $self->{guest_name} -L -Logfile $_guest_installation_log $self->{virt_install_command_line}\n", timeout => 600 / get_var('TIMEOUT_SCALE', 1));
+    #Use "screen" in the most compatible way, screen -t "title" -L -c "configuration file" "turn on output logging" "command to run".
```

Comment:
Can the screen command be made easier to read? BTW, for values, you can use <value>. May man pages use this way.

Reply:
'screen' is being used in its most common way. I do not know what you refer to. Could you help make your comment clearer and specific ?

Reply:
This part in comment:
screen -t "title" -L -c "configuration file" "turn on output logging" "command to run"

Reply:
Done